CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.13.0
+* Feature - Implemented automated image cleanup
+* Enhancement - Add credential caching for ECR
+* Enhancement - Add support for security-opt=no-new-privileges
+
```

Comment:
Can you add "Bug - Fixed a potential deadlock in `dockerstate`." (from [this commit](https://github.com/aws/amazon-ecs-agent/pull/543/commits/f7999e25a2931fd0105c75f95cea8ce072b002c2))?